acvm/src/pwg/mod.rs

```diff
@@ -376,17 +350,6 @@ mod tests {
         let w_z = Witness(4);
         let w_z_inverse = Witness(5);
         let opcodes = vec![
-            Opcode::Oracle(OracleData {
```

Comment:
This test is no longer testing what it says it is. We should rework this.

Reply:
I think we can delete this test entirely now